provision/roles/output_writer/tasks/main.yml

```diff
@@ -85,6 +88,8 @@
       os_volume_res: "{{ async_outputs_os_volume }}"
       os_heat_res: "{{ async_outputs_os_heat }}"
       aws_ec2_res: "{{ async_outputs_aws_ec2 }}"
+      aws_cfn_res: "{{ async_outputs_aws_cfn }}"
```

Comment:
Yes we need to add them.
Moreover , the async == true output is different from the async == false output . So have to write a custom filter to parse it.